### DISCUSSION TITLE
Typescript error with Tooltip's triggerType property

### DISCUSSION THREAD

**Discussion by jkiss** [heroui-inc/heroui#2518] [Q&A] [answered]
the doc has `triggerType` prop, but I get error in code:

![image](https://github.com/nextui-org/nextui/assets/3231326/67bd433d-40d4-4464-9ea6-1adef97d91b5)
![image](https://github.com/nextui-org/nextui/assets/3231326/e000e545-e12c-4471-8e9e-174f9b641723)

by the way, `triggerType` means trigger behavior right? like `click` or `hover`?

**Comment by wingkwong** [comment]
@jkiss can you also provide which version you are using?

**Reply by jkiss** [reply]
v2.2.10

![image](https://github.com/nextui-org/nextui/assets/3231326/bf2edcb5-c012-4268-9f7d-17c011387990)

**Comment by wingkwong** [comment, marked as answer]
Since it is for `popover` but I didn't see it is being used in tooltip. I've created a PR [here](https://github.com/nextui-org/nextui/pull/2520) to remove from the doc.

**Reply by jkiss** [reply]
🤣  ok!
